### PR TITLE
events page format from joseph

### PR DESCRIPTION
**Pull request by codetricity**

### CODE REVIEW

File: news/events.qmd [hunk 1]
```diff
@@ -7,18 +7,52 @@ format:
     css: events.css   
 ---
 
-## Upcoming Events
+  ## Upcoming Events
```
Comment (jcasman) [inline news/events.qmd:10]: Delete space at beginning of line, does not show up as header

**Review by jcasman — CHANGES_REQUESTED** [review]
Please make sure header displays properly (line 10)

**Comment by codetricity** [comment]
> Please make sure header displays properly (line 10)

I deleted the two spaces

File: news/events.qmd (hunk 1 above)
Comment (codetricity) [reply news/events.qmd]: image

**Pull request merged**